src/server/game/Grids/Notifiers/GridNotifiers.h

```diff
@@ -800,20 +823,22 @@ namespace Acore
     {
     public:
         AnyUnfriendlyUnitInObjectRangeCheck(WorldObject const* obj, Unit const* funit, float range) : i_obj(obj), i_funit(funit), i_range(range) {}
-        bool operator()(Unit* u)
+
+        bool operator()(Unit* u) const
         {
-            if (u->IsAlive() && i_obj->IsWithinDistInMap(u, i_range) && !i_funit->IsFriendlyTo(u) &&
-                    (i_funit->GetTypeId() != TYPEID_UNIT || !i_funit->ToCreature()->IsAvoidingAOE())) // pussywizard
```

Comment:
Why removed that?

Reply:
This part i just couldnt make sense at all with why it was present. Is there anything in game that would reference this situation of: being alive and indistance and friendly with Not being a Unit or avoid aoe? Or am i misreading that block entirely?

Reply:
Being alive, in distance, not friendly and (being a player or not being an aoe avoiding creature). So if creature avoids Aoe, then it should not be put in the list.
It's here for some purpose and I won't remove it cause it might break something.

Reply:
Like i mentioned before, im just not able to see where this was put into use. Granted jt seems exceedingly specific, but without a reference for what it is used in particular, im just not certain if this check was still being put into use with as much as the core has evolved. But if you prefer it to be kept that wouldnt be a issue, just let me know what you prefer and the readjustment will be put back in.

Reply:
Yes - restore that code.